pr_check.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="image-builder"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/image-builder-frontend"
```

Comment:
Out of curiosity, does this container have to be built manually with the build_deploy.sh script? Or will this end up in a GitHub Action?

Reply:
It should be handled by the @app-sre-bot you can see the results of it already in the checks here as `ci.ext.devshift.net PR build` :)

Reply:
yeah, it's automatically built by jenkins :) `build_deploy.sh` on merge to main and `pr_check.sh` on every PR push.